engine/src/faction_generic.cpp

```diff
@@ -37,6 +43,7 @@ void Faction::ParseAllies( unsigned int thisfaction )
         tempvec[faction[i].stats.index].stats = tmp;
     }
     faction.swap( tempvec );
+    }
```

Comment:
I think this `}` should go after line 34.

Reply:
Which } are you talking about?

Reply:
The one after `faction.swap( tempvec );`

Reply:
Agreed, unless we have a bug hiding here due to missing `{` :)

Reply:
I fixed that one up, as suggested